.github/workflows/spelling.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       - name: Run check
         uses: codespell-project/actions-codespell@master
         with:
-          skip: ./src/control/vendor,./.git
+          skip: ./src/control/vendor,./.git,./src/rdb/raft
```

Comment:
src/rdb/raft is a git submodule which shouldn't be checked out so shouldn't need excluding.